.github/labeler.yml

```diff
@@ -69,11 +69,13 @@ modifies/go:
   - changed-files:
       - any-glob-to-any-file:
           - "**/*.go"
+          - "**/*.tmpl"
```

Comment:
Isn't `tmpl` rather frontend?

Reply:
There is actually a `modifies/template` label above so I just removed it again.